SPECS/ruby/ruby.spec

```diff
@@ -82,8 +82,8 @@ autoconf
         --with-vendorarchdir=%{_libdir}/ruby/vendor_ruby \
         --with-rubyhdrdir=%{_includedir} \
         --with-rubyarchhdrdir=%{_includedir} \
-        --with-sitearchhdrdir=%{_prefix}/local/%{_lib}/ruby/site_ruby/$(_arch) \
-        --with-vendorarchhdrdir=%{_libdir}/ruby/vendor_ruby/$(_arch) \
+        --with-sitearchhdrdir=%{_prefix}/local/%{_lib}/ruby/site_ruby/$(uname -m) \
```

Comment:
Should fix error seen during build: /var/tmp/rpm-tmp.xdGUy3: line 48: _arch: command not found